apps/sage_intacct/views.py

```
@@ -244,3 +247,99 @@ def post(self, request: Request, *args, **kwargs) -> Response:
+        """
+        Exchange refresh token for code
+        """
+        payload = {
+            'grant_type': 'authorization_code',
+            'code': code,
+            'redirect_uri': redirect_uri,
+            'client_id': settings.INTACCT_CLIENT_ID,
+            'client_secret': settings.INTACCT_CLIENT_SECRET
+        }
+
+        response = requests.post(settings.INTACCT_TOKEN_URI, data=payload)
```

Ruff (0.14.4)

Probable use of `requests` call without timeout

(S113)